lib/cli-update-connection.ts

```diff
@@ -119,7 +119,7 @@ export default function CliUpdateConnection() {
             };
 
             client.gateways.patchDatasource(settings.collection, settings.workspace, datasource.gatewayId, datasource.id, delta, (err, patchResult) => {
-                if (err) {
```

Comment:
If the error is a Type error, this would be more proper to actually check `typeof callback === 'function'`.  As is any truthy value would attempt to be executed.